org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4whole/SimpleGUI.java

```diff
@@ -776,7 +779,9 @@ private Runner doOpen() {
 	private Runner doBuiltin() {
 		if (wrap)
 			return wrapMe();
-		File file = OurDialog.askFile(true, alloyHome() + fs + "models", ".als", ".als files");
+		File file = OurDialog.askFile(true, alloyHome() + fs + "models", new String[] {
+				".als", ".md"
+		}, "Alloy (.als) or Markdown files");
```

Comment:
Can we extract this string into a constant and instead of copy-pasting reuse it here and on line 770?

Reply:
I minimize the structural changes but I agree, bugged me as well.